Card/Event/Clearing.ts

```diff
@@ -0,0 +1,24 @@
+import { isoly } from "isoly"
+import { isly } from "isly"
+import { Amount } from "../../Amount"
+
+export interface Clearing {
+	type: Clearing.Type
+	created: isoly.DateTime
+	total: Amount
+	net: Amount
+	fee: Amount
+	charge?: Amount
```

Comment:
We could go with currency property and number so we don't say the currency 4 times.

Reply:
What about this?